librz/core/agraph.c

```diff
@@ -5,6 +5,7 @@
 #include <rz_util/rz_graph_drawable.h>
 #include <ctype.h>
 #include <limits.h>
+#include "private.h"
```

Comment:
This header name feels wrong.

Reply:
suggestions? It is used also in other modules so far e.g. io_private.h , bin/i/private.h

Reply:
`io_private` seems much better